AdminWebsite/AdminWebsite/Helper/DateListMapper.cs

```diff
@@ -21,10 +21,11 @@ public static List<DateTime> GetListOfWorkingDates(string startDate, string endD
 
         public static List<DateTime> GetListOfWorkingDates(DateTime startDate, DateTime endDate)
         {
+
             var dates = new List<DateTime>();
             var nextDate = startDate.AddDays(1);
 
-            for (var dt = nextDate; dt <= endDate; dt = dt.AddDays(1))
+            for (var dt = nextDate; dt.Date <= endDate.Date; dt = dt.AddDays(1))
```

Comment:
Should be both start and end date inclusive. No need to add a day to the end date

Reply:
The first booking is done in normal way, so I pass only clones dates, it should be like that